packages/account-v2/src/components/SignupWizard/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import ReactModal from 'react-modal';
+import { StandaloneXmarkBoldIcon } from '@deriv/quill-icons';
+import { useSignupWizardContext } from '../../context/SignupWizardContext';
+import Actions from './Actions';
+import { CUSTOM_STYLES } from './helpers';
+import './index.scss';
+
+const SignupWizard: React.FC = () => {
+    const {
+        currentStep,
+        helpers: { goToNextStep },
+        setIsWizardOpen,
+    } = useSignupWizardContext();
+    useEffect(() => {
+        ReactModal.setAppElement('#v2_modal_root');
+    }, []);
+    return (
+        <ReactModal
+            isOpen
+            onRequestClose={() => setIsWizardOpen(false)}
+            shouldCloseOnOverlayClick
+            style={CUSTOM_STYLES}
+        >
+            <div className='bg-background-primary-base h-[71.7rem] w-[104rem] rounded-800 flex overflow-hidden'>
+                <div className='min-w-[25.6rem] bg-[#f2f3f4] p-800'>Timeline</div>
```

Comment:
Please add `#f2f3f4` in the tailwind config file.  Also width 100% can be replaced by `w-full`